platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/network/connectivity/InboundConnectionHandler.java

```diff
@@ -139,31 +109,48 @@ public void handle(final Socket clientSocket) {
             newConnectionConsumer.accept(sc);
         } catch (final InterruptedException e) {
             Thread.currentThread().interrupt();
-            String formattedException = NetworkUtils.formatException(e);
+            final String formattedException = NetworkUtils.formatException(e);
             logger.warn(
                     SOCKET_EXCEPTIONS.getMarker(),
                     "Inbound connection from {} to {} was interrupted: {}",
-                    otherId == null ? "unknown" : otherId,
+                    "unknown",
                     selfId,
                     formattedException);
-            NetworkUtils.close(dis, dos, clientSocket);
+            NetworkUtils.close(clientSocket);
         } catch (final IOException e) {
-            String formattedException = NetworkUtils.formatException(e);
+            final String formattedException = NetworkUtils.formatException(e);
             socketExceptionLogger.warn(
                     SOCKET_EXCEPTIONS.getMarker(),
                     "Inbound connection from {} to {} had IOException: {}",
-                    otherId == null ? "unknown" : otherId,
+                    "unknown",
                     selfId,
                     formattedException);
-            NetworkUtils.close(dis, dos, clientSocket);
+            NetworkUtils.close(clientSocket);
         } catch (final RuntimeException e) {
             logger.error(
                     EXCEPTION.getMarker(),
                     "Inbound connection error, remote IP: {}\n" + "Time from accept to exception: {} ms",
                     clientSocket.getInetAddress().toString(),
                     acceptTime == 0 ? "N/A" : (System.currentTimeMillis() - acceptTime),
                     e);
-            NetworkUtils.close(dis, dos, clientSocket);
+            NetworkUtils.close(clientSocket);
+        }
+    }
+
+    private PeerInfo getConnectedPeer(@NonNull final SSLSocket sslSocket, @NonNull final List<PeerInfo> peers) {
+        final PeerInfo peer = Utilities.validateTLSPeer(sslSocket, peers);
+        if (peer == null) {
```

Comment:
add logging for null peer, include cert info

Reply:
done